scripts/script.py

```diff
@@ -272,6 +306,8 @@ def main():
         modifyFile(modPathCountlySessionsInternal, objectOfComModification, 'mod')
         modifyFile(modPathCountlyUserProfileInternal, objectOfComModification, 'mod')
         modifyFile(modPathCountlyViewsInternal, objectOfComModification, 'mod')
+        modifyFile(modPathExampleYaml, objectOfComModification, 'mod')
+        update_package("../example/lib/", 'package:countly_flutter/', 'package:countly_flutter_np/')
```

Comment:
I think by adding this line, we can also remove the explicit updating of import/export in SDK files for np.
update_package("../lib/", 'package:countly_flutter/', 'package:countly_flutter_np/')

Reply:
great point 👍  I currently scan first 30 lines of each file. should we increase that?

Reply:
done